crm_facebook_leads/models/crm_lead.py

```diff
@@ -11,7 +11,7 @@
 class CrmLead(models.Model):
     _inherit = 'crm.lead'
 
-    id_facebook_lead = fields.Char(readonly=True)
+    id_facebook_lead = fields.Char(string="Facebook Lead ID", readonly=True)
     facebook_page_id = fields.Many2one(
         'crm.facebook.page', related='facebook_form_id.page_id', store=True)
     facebook_form_id = fields.Many2one('crm.facebook.form', readonly=True)
```

Comment:
The fields `facebook_ad_id` and `facebook_campaign_id` are related to a field that is already in the `crm.lead` model, also, they are not being shown in the views, and I don't see any use of these fields, can you check and remove them if necessary.